packages/deb/build.go

```diff
@@ -85,7 +85,7 @@ func (ss *stringList) Set(v string) error {
 }
 
 var (
-	architectures = stringList{"amd64", "arm", "arm64", "ppc64le", "s390x"}
+	architectures = stringList{"amd64", "arm64", "ppc64le", "s390x"}
```

Comment:
@kubernetes/release-managers I'm wondering if we should make that change only affect v1.27+

I'm not completely sure how, but removing the platform for all supported branches seems wrong to me. Thoughts on that?

Reply:
I think the plan was to remove it from 1.28 in an orderly fashion, but this new go1.20 bug caused urgency

Reply:
But it also caused some issues with OBS, since it doesn't support cross-building (thus requring an arm32 OS)

Reply:
Yes so we can decide if we want to drop if from all branches, because we also use the latest go 1.20 for them which caused the linker issue.

Reply:
I misremembered, it was only Stream 8 that had some issues - it worked OK with `armv7l` on Ubuntu 20.04

Reply:
We will need to update the images for the other branches, right? if so i think we should keep that

Reply:
Yep, see https://github.com/kubernetes/release/pull/2960#issuecomment-1467724583